### ISSUE TITLE
Enhance: open exit

### ISSUE THREAD

**Issue by Nyyrazzilyss** [torilmud/issues#728]
Tatika 'OH! I was trying "open east" and that wasn't working'

Perhaps 'open direction' to open a visible exit in that direction, if it exists?

**Comment by Inames** [comment]
I feel like this would take away from the immersion and creativity that can be expressed in zone design. I know some MUD's do this, but it's not hard to learn to open an exit via interracting with something that's actually there... The helpfiles and newbie tutorial area in scardale need some revamping for sure though.

**Comment by shevarash** [comment]
I agree with Inames about immersion. However, I've been typing things like `open east` since I started on the MUD 25 years ago, and I _still_ do it, so maybe it's time.

**Comment by Nyyrazzilyss** [comment]
I have no real preference. I make that mistake myself on occasion: Usually typing open (direction), then immediately followed by 'exits' to find out what the exit name is.

I've no idea how much just 'open direction' is present in other games, or how often it's an actual problem. I'd posted this after answering a question for someone on NHC asking how to open doors.

**Comment by vitaladsr** [comment]
> I agree with Inames about immersion. However, I've been typing things like `open east` since I started on the MUD 25 years ago, and I _still_ do it, so maybe it's time.

While your messing with doors maybe you could do some cool stuff like create doors too tall for some chars to open, or too heavy haha .. or conversely two small for some classes to fit through .. would be an interesting arrow in the area designers quiver.

Annam
